site/src/components/DeploySettingsLayout/optionValue.ts

```diff
@@ -19,6 +22,26 @@ export function optionValue(option: ClibaseOption) {
       return Object.entries(option.value as Record<string, string>).map(
         ([key, value]) => `"${key}"->"${value}"`,
       );
+    case "Experiments": {
+      const experimentMap: Record<string, boolean> | undefined =
+        additionalValues?.reduce(
+          (acc, v) => {
+            return { ...acc, [v]: option.value.includes("*") ? true : false };
+          },
+          {} as Record<string, boolean>,
+        );
+
+      for (const v of option.value) {
+        if (
+          experimentMap &&
+          Object.prototype.hasOwnProperty.call(experimentMap, v)
```

Comment:
Even though [it doesn't show up in auto-complete](https://github.com/Microsoft/vscode/issues/54944), `experimentMap.hasOwnProperty` does still exist via the prototype chain, and doesn't need the `.call` syntax to be type-safe

Reply:
I'm actually using `call` here to get around an [eslint grumble](https://eslint.org/docs/latest/rules/no-prototype-builtins).
Looks like maybe I can use [`hasOwn`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/hasOwn) though...I'll try it out.

Reply:
Oh, yeah – ES Lint strikes again